src/app/config/config.c

```diff
@@ -2733,6 +2733,9 @@ list_enabled_modules(void)
 {
   printf("%s: %s\n", "relay", have_module_relay() ? "yes" : "no");
   printf("%s: %s\n", "dirauth", have_module_dirauth() ? "yes" : "no");
+  // We don't list dircache, because it cannot be enabled or disabled
+  // independently from relay.  Listing it here would proliferate
+  // tests in test_parseconf.c to no useful purpose.
```

Comment:
It's actually the parseconf script, not the unit tests :-)
```suggestion
  // test variants in test_parseconf.sh to no useful purpose.
```